cypress/TestCases/FireboltCertification/SecureStorage.feature

```diff
@@ -224,4 +224,96 @@ Feature: SecureStorage
             | Passing scope as number       | clear stored value with scope as number       |
             | Passing scope as null         | clear stored value with scope as null         |
             | Passing scope as boolean      | clear stored value with scope as boolean      |
-            | without scope                 | clear stored value without scope              |
\ No newline at end of file
+            | without scope                 | clear stored value without scope              |
```

Comment:
we should not have 2 given in a single tc

Reply:
if scope is device, we don;t need to set separately for secondary 3rd party app id